### PR TITLE
tikv: document backup.num-threads config

### PR DESCRIPTION
**Pull request by kennytm**
### What is changed, added or deleted? (Required)

Document the new TiKV configuration `backup.num-threads` introduced in tikv/tikv#8193.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -1139,7 +1139,7 @@ raftdb 相关配置项。
 
 ## import
 
-用于 import 相关的配置项。
+用于 Lightning 导入及 BR 恢复相关的配置项。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1142]:
```suggestion
用于 TiDB Lightning 导入及 BR 恢复相关的配置项。
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -1153,6 +1153,16 @@ raftdb 相关配置项。
 + 默认值：8
 + 最小值：1
 
+## backup
+
+用于 BR 备份相关的配置项。
+
+### `num-threads`
+
++ 处理备份工作线程数。
```
Comment (TomShawn) [inline tikv-configuration-file.md:1162]:
```suggestion
+ 处理备份的工作线程数。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -1153,6 +1153,16 @@ raftdb 相关配置项。
...
++ 默认值：CPU * 0.75，但最大为 32
```
Comment (TomShawn) [inline tikv-configuration-file.md:1163]: CPU 核数 * 0.75 吗？

Comment (kennytm) [reply tikv-configuration-file.md:1163]: @TomShawn yes. But the rest of the documents are all using `CPU * 0.8` (max-thread-count, high-concurrency, etc.), so this is for consistency.

**Pull request merged**